### PR TITLE
SDK-2027: adding non-latin support for session and supported documents

### PR DESCRIPTION
**Pull request by MichalMilewiczYoti**
adding non-latin support for session and supported documents

### CODE REVIEW

**Review by ShamariYoti — APPROVED** [review]
LGTM

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py [hunk 1]
```diff
@@ -94,10 +95,23 @@ def type_restriction(self):
         """
         return self.__type_restriction
 
+    @property
+    def allow_non_latin_documents(self):
+        """
+        Returns flag is non latin documents available.
```
Comment (echarrod) [inline yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:101]:
```suggestion
        Returns the flag for whether non-latin documents are allowed.
```

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py [hunk 2]
```diff
@@ -170,6 +185,26 @@ def with_blacklisted_document_types(self, document_types):
         self.__type_restriction = TypeRestriction(INCLUSION_BLACKLIST, document_types)
         return self
 
+    def allow_non_latin_documents(self):
+        """
+        Sets a True for allow non latin documents flag.
```
Comment (echarrod) [inline yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:190]:
```suggestion
        Sets a True value for "allow non-latin documents" flag.
```

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py [hunk 3]
```diff
@@ -170,6 +185,26 @@ def with_blacklisted_document_types(self, document_types):
...
+
+        :return: the builder
+        :rtype: OrthogonalRestrictionsFilterBuilder
+        """
+        self.__allow_non_latin_documents = True
+        return self
+
+    def disable_non_latin_documents(self):
+        """
+        Sets a False for allow non latin documents flag.
```
Comment (echarrod) [inline yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:200]:
```suggestion
        Sets a False value for "allow non-latin documents" flag.
```

**Review by echarrod — APPROVED** [review]
Looks good Michal, just a few documentation tweaks which would make it slightly easier to read for people without the context :slightly_smiling_face: :+1:

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py (hunk 1 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:101]: done

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py (hunk 2 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:190]: done

File: yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py (hunk 3 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py:200]: done

**Pull request merged**